pennylane_lightning/lightning_qubit/_measurements.py

```diff
@@ -411,7 +411,7 @@ def _process_single_shot(samples):
                         ).astype(int, copy=False)
                     else:
                         samples = self._measurement_lightning.generate_samples(
-                            len(wires), s
+                            list(wires), s
```

Comment:
With this change, we are calling the backend-specific generate_camples overload. Do we still have use for the backend-agnostic method in LightningQubit?

Reply:
> Do we still have use for the backend-agnostic method in LightningQubit?

No. I think we could port the backend-specific `generate_camples` in the other backends and make it the backend-agnostic method in a follow-up PR, but that's another story.

Reply:
Good point! Thanks!